styles.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+}
+
+.introduction {
+  display: flex;
+  flex-direction: column;
+  height: 604px;
+}
```

Comment:
- [ ] Please kindly add the background-image as we have it in figma design. 👍 
- [ ] Please carefully go through the design and give appropriate color and font-size. 👍